atomicapp/run.py

```diff
@@ -110,8 +113,12 @@ def _applyTemplate(self, data, component):
             except KeyError as ex:
                 name = ex.args[0]
                 logger.debug("Artifact contains unknown parameter %s, asking for it", name)
-                config[name] = self.utils.askFor(name, {"description": "Missing parameter '%s', provide the value or fix your %s" % (name, MAIN_FILE)})
+                try:
```

Comment:
yes this is added for cockpit integration. I want askFor method to give a user defined exception to cockpit.